### PR TITLE
Update feature support matrix to include Fantom and Polygon

### PR DESCRIPTION
**Pull request by PedroMD**
Proposing adding support for Fantom and Polygon to the feature support matrix. This PR will be included in the GGP (Graph Governance Proposal) to be created here https://snapshot.org/#/council.graphprotocol.eth.

This PR should be merged after Council approval. Afterward, the reference to the canonical GGP-ratified feature support matrix (end of the .md) will be updated.

### CODE REVIEW

**Comment by PedroMD** [comment]
@fordN I just noticed we also have https://github.com/graphprotocol/indexer/blob/main/docs/feature-support-matrix.md 
I there a reason why we have the matrix in two different .md files? Shouldn't we have a canonical one we can reference in GGPs to be ratified by the Council? 

And while I'm at it, is this the process we want to continue to follow? Would it make sense to, for instance, have the latest ratified feature matrix anchored on-chain in a DataEdge type of contract and indexed by the network subgraph or something like that?

**Comment by PedroMD** [comment]
Now that the Council [has ratified it](https://snapshot.org/#/council.graphprotocol.eth/proposal/0x56d75d710d3f90a96705f455628e1309666aa2eb280f5c4295291ca3728c3ed0), this one is ready to be merged. However, I believe we should have only 1 instance of this feature matrix for both Ethereum and Arbitrum and not multiple as we currently have (in `docs/feature-support-matrix.md` and `mainnet.md`). How we structure docs for different mainnet and testnet environments may also affect this. Payne is already proposing breaking it down into multiple files: https://github.com/graphprotocol/indexer/pull/660

@fordN , should we close this one for now and decide then how we want to structure these docs?

**Review by azf20 — CHANGES_REQUESTED** [review]
I think the live alias is `matic`?

File: docs/networks/mainnet.md [hunk 1]
```diff
@@ -86,25 +86,27 @@ valid from: 787
...
+| near:*                   | *            | Yes         | Yes          | No                | No                   | No               |
+| cosmos:*                 | *            | Yes         | Yes          | No                | No                   | No               |
+| arweave:*                | *            | Yes         | Yes          | No                | No                   | No               |
+| eip155:42161             | arbitrum-one | Yes         | Yes          | Yes               | Yes                  | Yes              |
+| eip155:42220             | celo         | Yes         | Yes          | Yes               | Yes                  | Yes              |
+| eip155:43114             | avalanche    | Yes         | Yes          | Yes               | Yes                  | Yes              |
+| eip155:250               | fantom       | Yes         | Yes          | Yes               | Yes                  | Yes              |
+| eip155:137               | polygon      | Yes         | Yes          | Yes               | Yes                  | Yes              |
```
[the remark this answers is not in the document]
Comment (PedroMD) [reply docs/networks/mainnet.md:106]: Oh, yes, it recently changed to `matic`. I'll update it

**Review by azf20 — APPROVED** [review]

**Pull request merged**